packages/jest-preset-mc-app/load-config.js

```diff
@@ -0,0 +1,48 @@
+const { cosmiconfigSync } = require('cosmiconfig');
+
+const moduleName = 'jest-mc-app';
```

Comment:
Should also be `pkg.name` almost of don't you want it to be `jest-preset-mc-app.yaml` eg?

Reply:
Yeah, I felt that the "preset" part was superfluous.

Reply:
Yeah maybe. However its also nice to have pkg name and cosmiconfig file names the same. Even if superfluous. Could bring clarity. Just a thought 😎